guide/src/python_from_rust.md

```diff
@@ -304,23 +304,28 @@ fn main() -> PyResult<()> {
 ```
 
 The example below shows:
-* how to load content of `app.py` and `utils/foo.py` at runtime
+* how to load content of `app.py` at runtime so that it sees its dependencies
+  automatically
 * how to call function `run()` (declared in `app.py`) that needs function
   imported from `utils/foo.py`
 
-It is recommended to use an absolute path to your Python files because then
-your binary can be run from anywhere.
+It is recommended to use an absolute paths because then your binary can be run
+from anywhere as long as your `app.py` is in expected directory (in example
+expected directory is `/usr/share/python_app`).
```

Comment:
```suggestion
It is recommended to use absolute paths because then your binary can be run
from anywhere as long as your `app.py` is in the expected directory (in this example
that directory is `/usr/share/python_app`).
```